src/wallet/scriptpubkeyman.cpp

```diff
@@ -197,15 +197,15 @@ IsMineResult LegacyWalletIsMineInnerDONOTUSE(const LegacyDataSPKM& keystore, con
 
 } // namespace
 
-isminetype LegacyDataSPKM::IsMine(const CScript& script) const
+bool LegacyDataSPKM::IsMine(const CScript& script) const
 {
     switch (LegacyWalletIsMineInnerDONOTUSE(*this, script, IsMineSigVersion::TOP)) {
     case IsMineResult::INVALID:
     case IsMineResult::NO:
-        return ISMINE_NO;
+        return false;
     case IsMineResult::WATCH_ONLY:
     case IsMineResult::SPENDABLE:
-        return ISMINE_SPENDABLE;
+        return true;
     }
```

Comment:
Note: this is okay because this is used only during wallet migration from legacy to descriptor.